.github/CODEOWNERS

```diff
@@ -148,6 +148,7 @@ pkg/batchperresourceattr/                                           @open-teleme
 pkg/batchpersignal/                                                 @open-telemetry/collector-contrib-approvers @jpkrohling
 pkg/experimentalmetricmetadata/                                     @open-telemetry/collector-contrib-approvers @rmfitzpatrick
 pkg/golden/                                                         @open-telemetry/collector-contrib-approvers @djaglowski @atoulme
+pkg/kafka/topic/                                                    @open-telemetry/collector-contrib-approvers @pavolloffay @MovieStoreGuy
```

Comment:
This line is added automatically from a directory having a `metadata.yaml` file that includes code owner information. There's more information [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#adding-new-components), and I believe you'll need to run `make generate-gh-issue-templates`, and also, if possible, update the README to have the format suggested there as well.

Reply:
That command didn't work for me, but I added that file and removed the line here.

Reply:
I run the command for you on 7764e24

Reply:
Thanks!

Reply:
There's some weird conflict going on.  `make crosslink` removes an extra link, but `make genoteltestbedcol` adds it back.  Any idea how to get past this?

Reply:
@tylerbenson I believe you don't have to add the replace to `builder-config.yaml` from `oteltestbedcol`, since `oteltestbedcol` does not include any of the kafka components